server/test/support/ResourceCreator.java

```diff
@@ -130,6 +130,14 @@ public ProgramModel insertActiveDisabledProgram(String name) {
     return ProgramBuilder.newDisabledActiveProgram(name).build();
   }
 
+  public ProgramModel insertActiveTiOnlyProgram(String name) {
```

Comment:
**non-blocking, nit**: it'd be nice if these names were the same format, so `[insert|new]ActiveTiOnlyProgram`